src/test/java/com/google/api/codegen/gapic/testdata/php/php_library.baseline

```diff
@@ -3144,27 +3184,48 @@ use \Google\Example\Library\V1\Book;
 use \Google\Example\Library\V1\SeriesUuid;
 use \Google\Example\Library\V1\Shelf;
 
-$libraryServiceClient = new LibraryServiceClient();
-
-// $name = 'Math';
-// $edition = 123;
-$shelf = new Shelf();
-$shelf->setName($name);
-$books = [];
-$seriesString = 'xyz3141592654';
-$seriesUuid = new SeriesUuid();
-$seriesUuid->setSeriesString($seriesString);
-
-try {
-    $response = $libraryServiceClient->publishSeries($shelf, $books, $seriesUuid, ['edition' => $edition]);
-    foreach ($response->getBookNames() as $title) {
-        printf("Title: %s" . PHP_EOL, $title);
+function samplePublishSeries($name, $edition)
+{
+    $libraryServiceClient = new LibraryServiceClient();
+
+    // $name = 'Math';
+    // $edition = 123;
+    $shelf = new Shelf();
+    $shelf->setName($name);
+    $books = [];
+    $seriesString = 'xyz3141592654';
+    $seriesUuid = new SeriesUuid();
+    $seriesUuid->setSeriesString($seriesString);
+
+    try {
+        $response = $libraryServiceClient->publishSeries($shelf, $books, $seriesUuid, ['edition' => $edition]);
+        foreach ($response->getBookNames() as $title) {
+            printf("Title: %s" . PHP_EOL, $title);
+        }
+        printf("That's all!" . PHP_EOL);
+        printf("series_uuid: %s" . PHP_EOL, $response->getSeriesUuid()->getSeriesBytes());
+    } finally {
+        $libraryServiceClient->close();
     }
-    printf("That's all!" . PHP_EOL);
-    printf("series_uuid: %s" . PHP_EOL, $response->getSeriesUuid()->getSeriesBytes());
-} finally {
-    $libraryServiceClient->close();
 }
+
+$opts = [
+    'name::',
+    'edition::',
+];
+
+$defaultOptions = [
+    'name' => 'Math',
+    'edition' => 123,
+];
+
+$options = getopt('', $opts);
+$options += $defaultOptions;
+
+$name = $options['name'];
+$edition = $options['edition'];
+
+samplePublishSeries($name, $edition);
```

Comment:
Purty 😍

Reply:
<3 the other samples w/o the empty options – so clean! simple function() calls – love it

Reply:
:smile: Thanks!